.travis.yml

```diff
@@ -3,11 +3,29 @@ language: python
 python:
   - 2.7
 install:
-  - pip install ansible-lint
-  - rvm install 2.2.4
+  - changes=$(git diff origin/master --name-only | grep "lib\|test" || true)
```

Comment:
Doesn't the ```|| true``` defeat the purpose?

Reply:
Nope -- without the `|| true` if there is no changes, the grep will throw an `exit 1` which marks the build as failed. Whereas with it, this will capture the output of the grep and prevent it failing so the `if` check can execute.